launcher/package/rpm/MultiMC5.spec

```diff
@@ -1,14 +1,20 @@
 Name:           MultiMC5
 Version:        1.4
-Release:        3%{?dist}
+Release:        4%{?dist}
 Summary:        A local install wrapper for MultiMC
 
 License:        ASL 2.0
 URL:            https://multimc.org
-BuildArch:      x86_64
```

Comment:
ExclusiveArch is the better alternative when trying to limit what arch a package is used on.

Since this only packages assets and a bash script it might be worthwhile to set this to noarch instead of removing it entirely